env.yml

```diff
@@ -0,0 +1,25 @@
+name: mle-dev
+channels:
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - ca-certificates=2020.10.14=0
+  - certifi=2020.11.8=py37h06a4308_0
+  - ld_impl_linux-64=2.33.1=h53a641e_7
+  - libedit=3.1.20191231=h14c3975_1
+  - libffi=3.3=he6710b0_2
+  - libgcc-ng=9.1.0=hdf63c60_0
+  - libstdcxx-ng=9.1.0=hdf63c60_0
+  - ncurses=6.2=he6710b0_1
+  - openssl=1.1.1h=h7b6447c_0
+  - pip=20.2.4=py37h06a4308_0
+  - python=3.7.9=h7579374_0
+  - readline=8.0=h7b6447c_0
+  - setuptools=50.3.1=py37h06a4308_1
+  - sqlite=3.33.0=h62c20be_0
+  - tk=8.6.10=hbc83047_0
+  - wheel=0.35.1=pyhd3eb1b0_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7b6447c_3
+prefix: /home/tiger0537/miniconda3/envs/mle-dev
```

Comment:
Try removing the prefix from ``env.yml`` file. You can make use of ``conda env export | grep -v "^prefix" > env.yml``.
Also, try to run the ``nonstandardcode.py`` file and resolve any error( ``Dependencies``) that occurs and retake the export using above mentioned command.